Print_A_Bot/controls/views.py

```diff
@@ -94,23 +93,21 @@ def post(self, request):
 
 class MoveBot(View):
     def get(self, request):
-        cmd_str = request.GET.get('cmd', None)
```

Comment:
changed out the direct calls to utils to pushing commands to redis where a 2nd process will pull them out and run the utilities
the premise is that this is how channels will be sending commands out and I wanted to keep support for the existing `GET` calls vs `websockets`